src/Microsoft.Extensions.Options/ConfigureNamedOptions.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Extensions.Options
     /// Implementation of IConfigureNamedOptions.
     /// </summary>
     /// <typeparam name="TOptions"></typeparam>
-    public class ConfigureNamedOptions<TOptions> : IConfigureNamedOptions<TOptions>, IConfigureOptions<TOptions> where TOptions : class
+    public class ConfigureNamedOptions<TOptions> : IConfigureNamedOptions<TOptions> where TOptions : class
```

Comment:
Isn't this a breaking change?

Reply:
IConfigureNamedOptions derives from IConfigureOptions, this is just cleanup